website/docs/modules/gather_sample_evidence.md

```diff
@@ -5,22 +5,26 @@ sidebar_position: 1
 slug: gse
 ---
 
+import { Highlight, HighlightOptionalArg } from "../../src/components/highlight.js"
+
+[WDL source code](https://github.com/broadinstitute/gatk-sv/blob/main/wdl/GatherSampleEvidence.wdl)
+
 Runs raw evidence collection on each sample with the following SV callers: 
-Manta, Wham, Scramble, and/or MELT. For guidance on pre-filtering prior to GatherSampleEvidence, 
-refer to the Sample Exclusion section.
+Manta, Wham, and Scramble, and collects raw SV evidence (PE/SR/RD/SD). For guidance on pre-filtering prior 
+to GatherSampleEvidence, refer to the [Input data](/docs/gs/inputs) section.
 
-The following diagram illustrates the downstream workflows of the `GatherSampleEvidence` workflow 
-in the recommended invocation order. You may refer to 
-[this diagram](https://github.com/broadinstitute/gatk-sv/blob/main/terra_pipeline_diagram.jpg) 
-for the overall recommended invocation order.
+:::note
+MELT is no longer supported as a raw caller. Please see [SV/CNV callers](/docs/gs/sv_callers) for more information.
+:::
 
+The following diagram illustrates the recommended invocation order:
```

Comment:
I find these helpful for providing context and avoiding another boilerplate line of text saying "This workflow should be run after X and before Y". For new users especially it would be awkward to have to go back to the diagram on another page to remind themselves of the order. There are also some cases that are non-linear, such as JoinRawCalls/SVConcordance, that isn't represented in the diagram or nav bar.